apps/src/applab/applab.js

```diff
@@ -680,7 +694,7 @@ Applab.init = function(config) {
     // should never be present on such levels, however some levels do
     // have levelHtml stored due to a previous bug. HTML set by levelbuilder
     // is stored in startHtml, not levelHtml.
-    if (config.level.hideDesignMode) {
+    if (Applab.reduxStore.getState().isDesignModeHidden) {
```

Comment:
Hmm.  Two thoughts.

* In this case we are both in the `init` method, and the "state" we're examining is a level property that's supposed to stay the same throughout the session.  For something like the level properties which are effectively a bundle of constants, would it be _better_ to freeze it and stick it on a global than to put it in redux?
* The other cases you're commented on do actually refer to changing state (`currentScreenId` in particular) and I agree that depending on the store in a non-React context takes some care.  I don't think it's actively harmful though - all of our non-React view code has to decide for itself when to re-render, even before implementing redux.  If anything it's easier now because we can subscribe to the store if we're worried about missing an update - something you can't do when depending on the DOM for state, as we do in some of our App Lab code.